docs/pipelines/library/service-endpoints.md

```diff
@@ -139,7 +139,7 @@ use one of the following techniques:
 
 - If you want to authorize any pipeline to use the service connection,
   go to Azure Pipelines, open the Settings page, select Service connections,
-  and enable the setting **Allow all pipelines to use this connection** option for the connection.
+  and enable the setting **Allow all pipelines to use this connection** option for the connection, found under the **Policies** tab.
```

Comment:
Thank you for your contribution, the service connections UI has been overhauled, you can enable the new UX from preview features. New experience has this setting as part of security config. We have to re-do this.
![image](https://user-images.githubusercontent.com/1370480/69743722-ee432100-1164-11ea-9509-6a20a0885cd1.png)

Reply:
@RoopeshNair I see. Thank you for that tip!